tests/user_commands/management/commands/subparser1.py

```diff
@@ -0,0 +1,12 @@
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        subparsers = parser.add_subparsers(title='subcommand')
```

Comment:
I think it should be `dest` instead of `title` to get error described in the ticket.